setup.py

```diff
@@ -5,7 +5,7 @@
 import subprocess
 try:
     # Python 2 backwards compat
-    from __builtin__ import raw_input as input
+    from builtins import raw_input as input
```

Comment:
as @GowthamShanmugam suggested
```suggestion
    from builtins import input
```

Reply:
Done